packages/database/prisma/migrations/20251027042954_add_headline_to_banner/migration.sql

```diff
@@ -0,0 +1,9 @@
+/*
+  Warnings:
+
+  - Added the required column `headline` to the `Banner` table without a default value. This is not possible if the table is not empty.
+  - Made the column `bannerImagePath` on table `Topic` required. This step will fail if there are existing NULL values in that column.
+
+*/
+-- AlterTable
+ALTER TABLE "public"."Banner" ADD COLUMN     "headline" TEXT NOT NULL;
```

Comment:
Please add default value to `headline` before remove default value from migration, this one will make sure that existing banner in database will have headline